src/admin/config.yml

```diff
@@ -158,16 +134,39 @@ collections:
       - label: Short Description
         name: shortDescription
         widget: markdown
-        hint: The short description is shown on the Initiatives page.
+        hint: The short description is shown on the Events page.
       - label: Preview Image
         name: previewImageUrl
         widget: image
         required: false
-        hint: The preview image is shown on the Initiatives page.
+        hint: The preview image is shown on the Events page.
       - label: Preview Image Alt Text
         name: previewImageAltText
         widget: string
         required: false
+  - name: recount
+    label: Recount
+    label_singular: Recount
+    folder: src/collections/recount
+    sortable_fields: [title, date]
+    create: true
+    fields:
```

Comment:
preview_path is missing for recount, when we had preview_path for news previously.

Reply:
News items can't be previewed. They are external links.